src/Groups/homomorphisms.jl

```diff
@@ -382,6 +382,20 @@ function Base.show(io::IO, A::AutomorphismGroup{T}) where T <: GAPGroup
   print(io, "Aut( "* String(GAP.Globals.StringView(A.G.X)) * " )")
 end
 
+"""
+  domain(A::AutomorphismGroup) -> Group
+
+Return the domain of this group of automorphisms.
+"""
+domain(A::AutomorphismGroup) = A.G
+
+"""
+  domain(f::AutomorphismGroupElem) -> Group
```

Comment:
Please make sure to indent the signatures line in docstrings with exactly four spaces: only these have the desired effect of activating code block formatting in Markdown, which will render the signatures as code, instead of as  plain text:
```suggestion
    domain(f::AutomorphismGroupElem) -> Group
```
Also, we don't use the `-> Group` notation to denote return types in most other places, I think? I am not against it, but I'd prefer if we were consistent about it. Perhaps this is something we all should discuss...

Reply:
The return type could be for instance a `GAPGroup` or a `HeckeGroup` or in the future a `...Group`. So group seemed appropriate. Though the name AutomorphismGroup seems to be a little too general or is the implementation intended to extend to represent the automorphism group of an object in any category?